lib/peddler/client.rb

```diff
@@ -51,6 +52,10 @@ def merchant_id
       @merchant_id ||= ENV['MWS_MERCHANT_ID']
     end
 
+    def auth_token
+      @auth_token ||= ENV['MWS_AUTH_TOKEN']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.